docs/mkdocs/tutorials/02_pipeline_building.ipynb

```diff
@@ -1,1415 +1,2341 @@
-            ]
-        },
-        {
-            "cell_type": "markdown",
-            "metadata": {},
-            "source": [
-                "## Recap"
-            ]
-        },
-        {
-            "cell_type": "markdown",
-            "metadata": {},
-            "source": [
-                "Data science workflows revolve around building and refining pipelines, i.e. series of processes that transform data into useful information/product. Traditionally, this is often manual and time-consuming work as data scientists (or other engineers) need to clean up messy development code and transform it into deployable scripts for the target system (e.g., Airflow). This tutorial demonstrated that LineaPy, having the complete development process stored in artifacts, can automate such code transformation, accelerating transition from development to production."
-            ]
-        },
-        {
-            "cell_type": "markdown",
-            "metadata": {},
-            "source": [
-                "<div class=\"admonition info\" style=\"background:#F0FAFA\">\n",
-                "    <p class=\"admonition-title\"><b>Info</b></p>\n",
-                "    <p>\n",
-                "        If you want to learn more about LineaPy's pipeline support, check out the project <a href=\"https://docs.lineapy.org/latest/guides/usage/pipeline-building/\">documentation</a>.\n",
-                "    </p>\n",
-                "</div>"
-            ]
-        }
-    ],
-    "metadata": {
-        "kernelspec": {
-            "display_name": "Python 3 (ipykernel)",
-            "language": "python",
-            "name": "python3"
-        },
-        "language_info": {
-            "codemirror_mode": {
-                "name": "ipython",
-                "version": 3
-            },
-            "file_extension": ".py",
-            "mimetype": "text/x-python",
-            "name": "python",
-            "nbconvert_exporter": "python",
-            "pygments_lexer": "ipython3",
-            "version": "3.10.3"
-        },
-        "vscode": {
-            "interpreter": {
-                "hash": "7b13a91c030603db07ad54ba4f9266c0e4bd5a63bc20907813ff665f2d224783"
-            }
-        }
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://bit.ly/3SJewuO)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Creating a Pipeline with LineaPy"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Table of Contents**\n",
+    "\n",
+    "- [Intro](#intro)\n",
+    "- [Demo](#demo)\n",
+    "    - [Exploration and Pre-processing](#exploration-and-pre-processing)\n",
+    "    - [Modeling](#modeling)\n",
+    "    - [Pipeline Building](#pipeline-building)\n",
+    "- [Additional Frameworks](#additional-frameworks)\n",
```

Comment:
“Additional Frameworks” -> “Orchestration Frameworks Supported”
Also some connector text from the last section to this section.
Something about how we showed you all the stuff that gets generated regardless of the frameworks. now let's show you the pipeline files we generate in supported orchestration frameworks.

Reply:
updated